README.md

```diff
@@ -42,3 +43,14 @@ Running the controller outside of the cluster depends on everything being in one
 1. Follow steps 1-5 for running the controller locally above
 2. `operator-sdk up local --namespace <your namespace> --enable-delve`
 3. Connect debugger to `127.0.0.1:2345` (see config in `.vscode/launch.json`)
+
+### Remove controller from your K8s/OS Cluster
+```sh
+# Delete all workspaces
+kc delete workspaces.workspace.che.eclipse.org --all-namespaces --all
```

Comment:
Just noticed -- do you also alias `kc=kubectl`? :smile:

Reply:
:-D I thought it's default alias ;)
Good catch